react/week1/todo_list/src/App.js

```diff
@@ -0,0 +1,32 @@
+import logo from './logo.svg';
+import './App.css';
+import Header from './Header';
+import ListItem from './ListItem';
+
+const todos = [
+  {
+    title: 'Get out of bed',
+    deadline: 'Wed Sep 13 2017'
+  },
+  {
+    title: 'Brush teeth',
+    deadline: 'Thu Sep 14 2017'
+  },
+  {
+    title: 'Eat breakfast',
+    deadline: 'Fri Sep 15 2017'
+  }
+]
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <ul>
+          {todos.map(todo => <ListItem title={todo.title} deadline={todo.deadline} />)}
```

Comment:
I don't know if this was covered in class, but lists in react should actually have a unique key. You can read more about those here: https://reactjs.org/docs/lists-and-keys.html#keys. If you would like to you can try to implement keys for your list.